web/src/context/installerL10n.jsx

```diff
@@ -49,37 +50,43 @@ function useInstallerL10n() {
 }
 
 /**
- * Current language according to Cockpit (in xx_XX format).
+ * Current language (in xx_XX format).
  *
- * It takes the language from the CockpitLang cookie.
+ * It takes the language from the AgamaLang cookie.
  *
  * @return {string|undefined} Undefined if language is not set.
  */
-function cockpitLanguage() {
+function agamaLanguage() {
   // language from cookie, empty string if not set (regexp taken from Cockpit)
   // https://github.com/cockpit-project/cockpit/blob/98a2e093c42ea8cd2431cf15c7ca0e44bb4ce3f1/pkg/shell/shell-modals.jsx#L91
-  const languageString = decodeURIComponent(document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*=\s*([^;]*).*$)|^.*$/, "$1"));
+  const languageString = decodeURIComponent(document.cookie.replace(/(?:(?:^|.*;\s*)AgamaLang\s*=\s*([^;]*).*$)|^.*$/, "$1"));
```

Comment:
I am not sure about the best practices for naming cookies. We used `agamaToken` for the token, following the example we found in the [Set-Cookie documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#session_cookie) (e.g., `sessionId`).

I would try to be consistent.